docs/preview/clients/overview.md

```diff
@@ -10,7 +10,7 @@ DuckDB is an in-process database system and offers client APIs (also known as 
 | [C]({% link docs/preview/clients/c/overview.md %})                              | The DuckDB team                                                                         | Primary      | [{{ site.current_duckdb_version }}]({% link install/index.html %}?version=stable&environment=cplusplus)           |
 | [Command Line Interface (CLI)]({% link docs/preview/clients/cli/overview.md %}) | The DuckDB team                                                                         | Primary      | [{{ site.current_duckdb_version }}]({% link install/index.html %}?version=stable&environment=cli)                 |
 | [Java (JDBC)]({% link docs/preview/clients/java.md %})                          | The DuckDB team                                                                         | Primary      | [{{ site.current_duckdb_java_short_version }}](https://central.sonatype.com/artifact/org.duckdb/duckdb_jdbc)                |
-| [Go]({% link docs/preview/clients/go.md %})                                     | [Marc Boeker](https://github.com/marcboeker) and the DuckDB team                        | Primary      | [{{ site.current_duckdb_go_version }}](https://github.com/marcboeker/go-duckdb?tab=readme-ov-file#go-sql-driver-for-duckdb) |
+| [Go]({% link docs/preview/clients/go.md %})                                     | [Marc Boeker](https://github.com/duckdb) and the DuckDB team                        | Primary      | [{{ site.current_duckdb_go_version }}](https://github.com/duckdb/duckdb-go?tab=readme-ov-file#go-sql-driver-for-duckdb) |
```

Comment:
Tip: If you want to link to a GitHub README, you can use this syntax:

https://github.com/duckdb/duckdb-go#readme